src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -11,103 +11,80 @@
 import frc.robot.utils.Utils;
 
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import com.ctre.phoenix.motorcontrol.ControlMode;
 import com.ctre.phoenix.motorcontrol.SensorCollection;
-import edu.wpi.first.wpilibj.drive.DifferentialDrive;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.util.Units;
+import edu.wpi.first.wpilibj.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj.kinematics.DifferentialDriveKinematics;
+import edu.wpi.first.wpilibj.kinematics.DifferentialDriveWheelSpeeds;
 
 public class DriveSubsystem extends SubsystemBase {
   // Need to use WPI_TalonFX so that DifferentialDrive will accept the motors.
   private final WPI_TalonFX leftFront = new WPI_TalonFX(FALCON_DRIVE_LEFT_FRONT_CAN_ADDR);
   private final WPI_TalonFX leftRear = new WPI_TalonFX(FALCON_DRIVE_LEFT_REAR_CAN_ADDR);
   private final WPI_TalonFX rightFront = new WPI_TalonFX(FALCON_DRIVE_RIGHT_FRONT_CAN_ADDR);
   private final WPI_TalonFX rightRear = new WPI_TalonFX(FALCON_DRIVE_RIGHT_REAR_CAN_ADDR);
-  private final DifferentialDrive drive;
 
   private final SensorCollection leftEnc = new SensorCollection(leftFront);
   private final SensorCollection rightEnc = new SensorCollection(rightFront);
 
+  private final DifferentialDriveKinematics kinematics = new DifferentialDriveKinematics(Units.inchesToMeters(26));
+
   private final boolean pidEnabled = false;
 
-  private final PIDController movePID;
-  private final PIDController turnPID;
+  /**
+   * @param ENC_TICKS_PER_FOOT The number of TalonFX encoder ticks per foot that
+   *                           the robot drives on 6" wheels. Used to input ft/s
+   *                           into WPI_TalonFX.set(VelocityControl, double
+   *                           value), which takes its value in encoder ticks per
+   *                           second.
+   * 
+   */
+  private final double ENC_TICKS_PER_FOOT = 1304;
 
   public DriveSubsystem() {
     // Slaves the left rear motor to the left front motor
     leftRear.follow(leftFront);
     rightRear.follow(rightFront);
-
-    // Initializes the 2 PIDControllers that will generate appropriate motor power
-    // values
-    movePID = new PIDController(1, 1, 1);
-    turnPID = new PIDController(1, 1, 1);
-
-    // Links both master-slave groups
-    drive = new DifferentialDrive(leftFront, rightFront);
-
-    drive.setDeadband(0);
   }
```

Comment:
Hmm, you don't ever initialize any of the PID constants.
At the very least have kP, kI, kD, kIz, and kFF constants which you then set on the motors using `config_kP()`, `config_kI()`, `config_kD()`, `config_IntegralZone()`, and `config_kF()`.

See [here](https://phoenix-documentation.readthedocs.io/en/latest/ch16_ClosedLoop.html#velocity-closed-loop-control-mode) for tips on how to start finding kP, kI, and kD, and [here](https://phoenix-documentation.readthedocs.io/en/latest/ch16_ClosedLoop.html#velocity-closed-loop-control-mode) to see how to calculate a starting value for kF